packages/cli/src/utils/build.js

```diff
@@ -322,10 +322,16 @@ const maybeNotifyAboutOutdated = () => {
   }
 };
 
-const build = async (zipPath, sourceZipPath, wdir) => {
-  wdir = wdir || process.cwd();
-  zipPath = zipPath || constants.BUILD_PATH;
-  sourceZipPath = sourceZipPath || constants.SOURCE_PATH;
+// TODO: use listr?
+// TODO: fix actually passing the opts around
+const build = async (
+  opts,
+  {
+    zipPath = constants.BUILD_PATH,
```

Comment:
this is a better pattern of defaults. `opts` are the command line flags (dumb paths & include maps).

Reply:
`opts` will replace calls to `global.argOpts` (such as [here]()) but I hadn't actually hooked that up yet.